libres/lib/include/ert/logging.hpp

```diff
@@ -0,0 +1,58 @@
+#ifndef RES_LOGGING_HPP
```

Comment:
Can you put moving this to a seperate commit?

Reply:
Mmmm... not following... 

It is not only moved but also slightly changed to make it #include'able in the Catch2-test (pybind11 header was not available there, nor needed). The remaining code is left in the original file (`python.hpp`), although that might perhaps be dropped?